src/components/Form/index.js

```diff
@@ -64,53 +55,80 @@ const FormSubmit = styled.div`
-    }
-`
-
 export default function Form({onSubmit, errorSending}) {
-    const [step, setStep] = useState('phone')
+    const [tab, setTab] = useState('phone')
     const { t } = useTranslation('form')
+    
+    const initialValues = {
+        personName: '',
+        phone: '',
+        mail: '',
+    }
+    
+    const onValidate = (values) => {
+        return triggerValidation(values, tab, t)
+    }
 
     const handleSubmitForm = (values, {setSubmitting}) => {
         if (onSubmit) {
             onSubmit(values, setSubmitting)
         }
     }
 
+    const onSelectTab = (tab) => setTab(tab)
+
     return (
         <FormCard>
             <FormCardHeader>
                 <p>{t('title')}</p>
-                {/* <p>{t('description')}</p> */}
+                <p>{t('description')}</p>
             </FormCardHeader>
-            <ButtonChoices>
-                {/* <Button disabled={isStepPhone(step)} onClick={() => setStep('phone')}>{t('buttons.sms.label')}</Button> */}
-                {/* <Button disabled={!isStepPhone(step)} onClick={() => setStep('mail')}>{t('buttons.mail.label')}</Button> */}
-                <p className="sms-disabled">L'option sms est momentanément désactivée. Elle sera rétablie le 1er avril 2020.</p>
-            </ButtonChoices>
-            <Formik validationSchema={schema} initialValues={initialValues} onSubmit={handleSubmitForm}>
+            <Formik initialValues={initialValues} validateOnBlur={false} validateOnChange={false} validate={onValidate} onSubmit={handleSubmitForm}>
                 {props => {
                 /* eslint-disable react/prop-types */
-                    const {isSubmitting, isValid, handleSubmit, dirty} = props
+                    const {isSubmitting, handleSubmit} = props
                     return (
-
                         <BootstrapForm onSubmit={handleSubmit} noValidate autoComplete='off'>
+                            <Tabs activeKey={tab} onSelect={onSelectTab}>
+                                <Tab eventKey="phone" title={t('buttons.sms.label')}>
+                                    <FormFields>
+                                        <Field
+                                            name="phone"
+                                            type="tel"
+                                            placeholder={t('phone.placeholder')} 
+                                            label={t('phone.label')} 
+                                            disabled={isSubmitting}
+                                            title={t('phone.title')} 
+                                        />
+                                    </FormFields>
+                                </Tab>
+                                <Tab eventKey="mail" title={t('buttons.mail.label')}>
+                                    <FormFields>
+                                        <Field
+                                            name="mail"
+                                            type="mail"
```

Comment:
```suggestion
                                            type="email"
```

a "mail" is a letter in english

Reply:
done